hbase_sql/src/main/java/com/splicemachine/serializer/SpliceKryoSerializerInstance.java

```diff
@@ -26,7 +26,7 @@
 public class SpliceKryoSerializerInstance extends KryoSerializerInstance {
 
     public SpliceKryoSerializerInstance(KryoSerializer ks) {
-        super(ks);
+        super(ks,false);
```

Comment:
@jleach4 : Please review this comment, so wen can proceed to the merge after that clear out.